Sources/NIO/ChannelPipeline.swift

```diff
@@ -446,9 +446,16 @@ public final class ChannelPipeline: ChannelInvoker {
         return promise.futureResult
     }
 
+    /// Returns a `ChannelHandlerContext` which matches.
+    ///
+    /// This skips head and tail (as these are internal and should not be accessible by the user.
+    ///
+    /// - parameters:
+    ///     - body: The predicate to execute per `ChannelHandlerContext` in the `ChannelPipeline`.
+    /// -returns: The `ChannelHandlerContext` that matches or `nil` if non did.
```

Comment:
We should note that this returns the *first* `ChannelHandlerContext` that matches, not the only one.

Reply:
done